### ISSUE TITLE
Adam optimiser not using the specified learning rate

### ISSUE THREAD

**Issue by rajathirumal** [AymenBOUGUERRA/Deep_Learning_For_Credit_Approval#1]
![image](https://github.com/AymenBOUGUERRA/Deep_Learning_For_Loan_Approval/assets/41321221/9f27e6cf-79c7-49e9-9426-5198fd72b128)

Since you have not used the Adam optimiser there is no need to create an instance for it. As you are not using it Keras will use the default parameters (learning_rate:0.001) for the Adam optimiser. 

```python 
optimizer = Adam( learning_rate = 0.000005)

model.compile(loss='binary_crossentropy',
              optimizer=optimizer ,
              metrics=['accuracy'])
